headers/types/dungeon_mode/dungeon.h

```diff
@@ -1678,134 +1682,10 @@ struct dungeon {
-    undefined field_0x37b7;
-    undefined field_0x37b8;
-    undefined field_0x37b9;
-    undefined field_0x37ba;
-    undefined field_0x37bb;
-    undefined field_0x37bc;
-    undefined field_0x37bd;
-    undefined field_0x37be;
-    undefined field_0x37bf;
-    undefined field_0x37c0;
-    undefined field_0x37c1;
-    undefined field_0x37c2;
-    undefined field_0x37c3;
-    undefined field_0x37c4;
-    undefined field_0x37c5;
-    undefined field_0x37c6;
-    undefined field_0x37c7;
-    undefined field_0x37c8;
-    undefined field_0x37c9;
-    undefined field_0x37ca;
-    undefined field_0x37cb;
-    undefined field_0x37cc;
-    undefined field_0x37cd;
-    undefined field_0x37ce;
-    undefined field_0x37cf;
-    undefined field_0x37d0;
-    undefined field_0x37d1;
-    undefined field_0x37d2;
-    undefined field_0x37d3;
-    undefined field_0x37d4;
-    undefined field_0x37d5;
-    undefined field_0x37d6;
-    undefined field_0x37d7;
-    undefined field_0x37d8;
-    undefined field_0x37d9;
-    undefined field_0x37da;
-    undefined field_0x37db;
-    undefined field_0x37dc;
-    undefined field_0x37dd;
-    undefined field_0x37de;
-    undefined field_0x37df;
-    undefined field_0x37e0;
-    undefined field_0x37e1;
-    undefined field_0x37e2;
-    undefined field_0x37e3;
-    undefined field_0x37e4;
-    undefined field_0x37e5;
-    undefined field_0x37e6;
-    undefined field_0x37e7;
-    undefined field_0x37e8;
-    undefined field_0x37e9;
-    undefined field_0x37ea;
-    undefined field_0x37eb;
-    undefined field_0x37ec;
-    undefined field_0x37ed;
-    undefined field_0x37ee;
-    undefined field_0x37ef;
-    undefined field_0x37f0;
-    undefined field_0x37f1;
-    undefined field_0x37f2;
-    undefined field_0x37f3;
-    undefined field_0x37f4;
-    undefined field_0x37f5;
-    undefined field_0x37f6;
-    undefined field_0x37f7;
-    undefined field_0x37f8;
-    undefined field_0x37f9;
-    undefined field_0x37fa;
-    undefined field_0x37fb;
-    undefined field_0x37fc;
-    undefined field_0x37fd;
-    undefined field_0x37fe;
-    undefined field_0x37ff;
-    undefined field_0x3800;
-    undefined field_0x3801;
-    undefined field_0x3802;
-    undefined field_0x3803;
-    undefined field_0x3804;
-    undefined field_0x3805;
-    undefined field_0x3806;
-    undefined field_0x3807;
-    undefined field_0x3808;
-    undefined field_0x3809;
-    undefined field_0x380a;
-    undefined field_0x380b;
-    undefined field_0x380c;
-    undefined field_0x380d;
-    undefined field_0x380e;
-    undefined field_0x380f;
-    undefined field_0x3810;
-    undefined field_0x3811;
-    undefined field_0x3812;
-    undefined field_0x3813;
+    // 0x3794: Table of monsters that can be spawned on the floor.
+    // The number of valid entries is stored in monster_spawn_table_length.
+    // The last 2 entries of the table are always Kecleon (0x17F) and the dummy (0x229).
+    struct monster_spawn_entry monster_spawn_table[16];
```

Comment:
I noticed that there was already an instance of this at 0x2C964. This one is used when spawning monsters, but the other one seems to be used during floor initialization. Should I call this `spawn_entries` and rename the other to something like `spawn_entries_master`?